ADMVEDU95/ADMVEDU95/Views/AudioDetailView.swift

```diff
@@ -8,120 +8,29 @@
-        addSubview(songNameLabel)
-        addSubview(artistNameLabel)
-        addSubview(albumNameLabel)
-        addSubview(playPauseButton)
-    }
-    
-    private func setupConstraints() {
-        setupAlbumImage()
-        setupArtistName()
-        setupSongName()
-        setupAlbumName()
-        setupplayPauseButton()
-    }
-    
-    private func setupAlbumImage() {
-        albumImageView.anchor(top: self.safeAreaLayoutGuide.topAnchor,
-                              paddingTop: Constants.smallInterval)
-        albumImageView.dimension(width: Constants.albumImageSize,
-                                 height: Constants.albumImageSize)
-        albumImageView.centerAnchor(centerX: self.centerXAnchor)
-    }
-    
-    private func setupArtistName() {
-        artistNameLabel.anchor(top: albumImageView.bottomAnchor,
-                               paddingTop: Constants.bigInterval,
-                               left: self.leftAnchor,
-                               paddingLeft: Constants.sideInterval,
-                               right: self.rightAnchor,
-                               paddingRight: Constants.sideInterval)
-        artistNameLabel.centerAnchor(centerX: self.centerXAnchor)
-    }
-    
-    private func setupSongName() {
-        songNameLabel.anchor(top: artistNameLabel.bottomAnchor,
-                             paddingTop: Constants.smallInterval,
-                             left: self.leftAnchor,
-                             paddingLeft: Constants.sideInterval,
-                             right: self.rightAnchor,
-                             paddingRight: Constants.sideInterval)
-        songNameLabel.centerAnchor(centerX: self.centerXAnchor)
-    }
-    
-    private func setupAlbumName() {
-        albumNameLabel.anchor(top: songNameLabel.bottomAnchor,
-                              paddingTop: Constants.smallInterval,
-                              left: self.leftAnchor,
-                              paddingLeft: Constants.sideInterval,
-                              right: self.rightAnchor,
-                              paddingRight: -Constants.sideInterval)
-        albumNameLabel.centerAnchor(centerX: self.centerXAnchor)
-    }
-    
-    private func setupplayPauseButton() {
-        playPauseButton.anchor(top: albumNameLabel.bottomAnchor,
-                               paddingTop: Constants.bigInterval)
-        playPauseButton.dimension(width: Constants.buttonSize,
-                                  height: Constants.buttonSize)
-        playPauseButton.centerAnchor(centerX: self.centerXAnchor)
-        playPauseButton.setBackgroundImage(UIImageView.playButtonImage, for: .normal)
+        commonInit()
     }
     
-    private func createLabel(numberOfLines: Int, textAlighment: NSTextAlignment) -> UILabel {
-        let label = UILabel()
-        label.numberOfLines = numberOfLines
-        label.textAlignment = textAlighment
-        return label
+    private func commonInit() {
```

Comment:
common init should be implemented in a base class to avoid duplicated code for views with xibs